.changeset/lucky-spoons-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/identity-auth-frontend': patch
```

Comment:
a peerdep change [should be a major version](https://github.com/guardian/recommendations/blob/main/npm-packages.md#changes-to-peerdependencies-ranges-are-breaking)

```suggestion
'@guardian/identity-auth-frontend': major
```